1077/en.md

```diff
@@ -2,7 +2,9 @@
 
 
 ## Solution
-The Number of lattice points lie on the segment **A(x1, y1) -> B(x2, y2)** is same for segment **A(0, 0) -> B(x2 - x1, y2 - y1)** (co-ordinate translation doesn't change the relative distance of lattice points from each other) and so for segment **A(0, 0) -> B(|x2 - x1|, |y2 - y1|)** (sign (+/-) only let's us know in which quadrant the segment will fall. Number of lattice points is length dependent not quadrant dependent).
+The Number of lattice points lie on the segment **A(x1, y1) -> B(x2, y2)** is same for segment **A(0, 0) -> B(x2 - x1, y2 - y1)** because co-ordinate translation doesn't change the relative distance of lattice points from each other.
```

Comment:
~~The Number of lattice points lie~~

The number of lattice points lying on

Reply:
done!

Reply:
> The number of lattice points **lying** on

You missed one.

Reply:
We can merge right after this one. @Talha-Taki002

Reply:
I am so sorry for my overlooking mistake

Reply:
Yeah no worries. It's not a biggie. I was just nitpicking there.